integration-test/src/test/java/com/datastrato/gravitino/integration/test/util/spark/SparkTableInfoChecker.java

```diff
@@ -61,20 +78,47 @@ public SparkTableInfoChecker withIdentifyPartition(List<String> partitionColumns
     return this;
   }
 
-  public SparkTableInfoChecker withBucket(int bucketNum, List<String> bucketColumns) {
+  public SparkTableInfoChecker withBucketPartition(int bucketNum, List<String> bucketColumns) {
     Transform bucketTransform = Expressions.bucket(bucketNum, bucketColumns.toArray(new String[0]));
-    this.expectedTableInfo.setBucket(bucketTransform);
-    this.checkFields.add(CheckField.BUCKET);
+    this.expectedTableInfo.addPartition(bucketTransform);
+    this.checkFields.add(CheckField.PARTITION);
     return this;
   }
 
-  public SparkTableInfoChecker withBucket(
-      int bucketNum, List<String> bucketColumns, List<String> sortColumns) {
-    Transform sortBucketTransform =
-        SparkTransformConverter.createSortBucketTransform(
-            bucketNum, bucketColumns.toArray(new String[0]), sortColumns.toArray(new String[0]));
-    this.expectedTableInfo.setBucket(sortBucketTransform);
-    this.checkFields.add(CheckField.BUCKET);
+  public SparkTableInfoChecker withHourPartition(String partitionColumn) {
+    Transform hourTransform = Expressions.hours(partitionColumn);
+    this.expectedTableInfo.addPartition(hourTransform);
+    this.checkFields.add(CheckField.PARTITION);
```

Comment:
could you change `checkFields` from `list` to `set`?

Reply:
updated.